sass/style.css

```diff
@@ -181,12 +220,34 @@ h3.city {
   text-transform: uppercase;
 }
 
+.locations-container {
+  padding: 0 !important;
+  margin: 0 !important;
+}
+
 .locations-list-header {
+  padding: 0 !important;
+  margin: 0 !important;
   border-bottom: 1px solid #D4D4D4;
 }
 
+.locations-list-header > * {
+  padding: 1rem 2rem;
+  white-space: 
+}
+
+.locations-list-header #list-stats {
+  font-size: 16px;
+  font-weight: 600;
+}
+
 .location {
   border-bottom: 1px solid #D4D4D4;
+  padding: 1.5rem 2rem !important;
+}
+
+.location .font-weight-bold {
+  font-weight: 600 !important;
```

Comment:
Get rid of the font-weight-bold class on the element, add a semantic class instead and style that.